cockatrice/src/tab_supervisor.cpp

```diff
@@ -446,11 +446,27 @@ void TabSupervisor::processGameEventContainer(const GameEventContainer &cont)
 
 void TabSupervisor::processUserMessageEvent(const Event_UserMessage &event)
 {
-    TabMessage *tab = messageTabs.value(QString::fromStdString(event.sender_name()));
+    QString senderName = QString::fromStdString(event.sender_name());
+
+    UserListTWI *twi = tabUserLists->getAllUsersList()->getUsers().value(senderName);
+
+    ServerInfo_User Self;
+
+    if (twi)
+        Self = twi->getUserInfo();
+    else
+        Self.set_name(senderName.toStdString());
```

Comment:
If twi is null, then this ServerInfo will have the default flags - so I don't think you can make a sensible decision on blocking it.

Also if it's populated from `senderName`, I'd expect this to be `sender` - this being `Self` makes me think it's referring to the user running cockatrice.

Variables should also be `camelCase`, not `PascalCase`